.studio/tooling

```diff
@@ -105,6 +105,11 @@ function channel_contains_commit() {
     else
         echo -e '   current: \033[0;91m✗\033[0m'
     fi
+
+    oldest_tag=$(git tag --contains "$commit" | head -1)
```

Comment:
I'm wondering if we want to filter this on tags that match our release numbers: `20200408145843`